generate/scope_generator.go

```diff
@@ -98,8 +104,12 @@ func GenTokenScope() {
 	defer f.Close()
 
 	err = tokenTemplate.Execute(f, struct {
-		Scopes []ScopeName
-	}{Scopes: scopes})
+		Scopes        []ScopeName
```

Comment:
Ah, I think the reason why the linter reported error is probably that you didn't have `pre-commit` installed locally. This error might be buried in other linter errors from the generated code, but one error among them is that this line of code wasn't being `gofmt -ed with -s`. Could you try install `pre-commit` locally?

Reply:
Hmm, pre-commit definitely kicks of its list of checks every time I commit. I do use a custom dev container, but my Dockerfile does run `pip3 install pre-commit`